demo/can-ndjson-stream.html

```diff
@@ -0,0 +1,38 @@
+<script src="../node_modules/steal/steal.js" main="@empty">
+var ndjsonStream = require("can-ndjson-stream");
+
+function streamerr(e) {
+  console.warn("Stream error");
+  console.warn(e);
+}
+
+fetch("/api").then( function(response) { 
+  return ndjsonStream(response.body);
+ }).then(function(todosStream) { 
+  var reader = todosStream.getReader();
+
+  reader.read().then(function read(result) {
+    if (result.value.user === '17437675') {
+      reader.cancel('Matched userId');
+      return;
+    }
+
+    if (result.done) {
+      render("Done.");
+      return;
+    }
+
+    render(result.value);   
+    reader.read().then(read, streamerr);
+
+  }, streamerr);
+ });
+
+var counter = 0;
+
+render = val => {
```

Comment:
It is OK for the demo and documentation bc any browser that supports fetch + readable streams also supports those ES6 features (I looked into this), but the library code and tests are in ES5.